setup.cfg

```diff
@@ -1,2 +1,8 @@
 [wheel]
-universal = 1
\ No newline at end of file
+universal = 1
+
+[flake8]
+exclude = examples, docs
+
+[pep257]
```

Comment:
We should probably switch to pydocstyle (the renamed version), but we can do that in a different PR.